### PR TITLE
application-test.yaml updated for testcases  success

### PR DESCRIPTION
**Pull request by denduluriramasaimohanakhil**
dev:
  lint:211
  coverage:85%
redirecting:
  lint:209
  coverage:85%

Testcases resolved

### CODE REVIEW

File: src/test/resources/application-test.yaml [hunk 1]
```diff
@@ -1,6 +1,16 @@
 keycloak:
   issuer-uri: ${KEYCLOAK_URL_AUTH:https://keycloak-tsplatform.techsophy.com}/auth/realms/
   realm.name: ${KEYCLOAK_REALM_NAME:techsophy-platform}
+  client.id: camunda-identity-service
+  auth-server-url: https://keycloak-tsplatform.techsophy.com/auth
+  multi-realm:
+    username: venkataramana
```
Comment (VaibhavTechsophy) [inline src/test/resources/application-test.yaml]: these properties should not be committed to the source code

File: src/test/resources/application-test.yaml [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+    password: Tech@4567
```
Comment (VaibhavTechsophy) [inline src/test/resources/application-test.yaml]: these values should not be committed to code

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]

File: src/test/resources/application-test.yaml (hunk 1 above)
Comment (denduluriramasaimohanakhil) [reply src/test/resources/application-test.yaml]: yes vaibhav giving like this ${MULTI_REALM_USERNAME:test}

**Review by VaibhavTechsophy — APPROVED** [review]

**Comment by VaibhavTechsophy** [comment]
pls deploy

File: src/test/resources/application-test.yaml (hunk 2 above)
Comment (denduluriramasaimohanakhil) [reply src/test/resources/application-test.yaml]: yes vaibhav giving dummy testcase values

**Pull request merged**